fast64_internal/f3d/f3d_gbi.py

```diff
@@ -3452,7 +3454,10 @@ def to_binary(self, f3d, segments):
 
     def to_c(self, static=True):
         if static:
-            return "gsSPDisplayList(" + self.displayList.name + ")"
+            if bpy.context.scene.fast64.oot.shipOfHarkinianCompatible and self.displayList.name.startswith('0x'):
+                return "gsSPDisplayList(" + self.displayList.name + ' | 1)'
```

Comment:
I really don't understand how this is supposed to change endianness, but if you're looking for segmented addresses, `gsSPMatrix` should also be taken into account.